samples/c_cxx/opschema_lib_use/main.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Samples don't get run as part of CI. If you want this functionality tested, it should be part of our unit tests.

Reply:
Thanks. Yes, I was wondering if this should be a part of the CI or not. At the moment, its main purpose is to act as a sample for users who want to link in the library, which is why I decided to leave it as a sample. If needed, we can turn it into a CI unit test.